### ISSUE TITLE
Failed to contruct `Headers` on Samsung TV UE55MU6102

### ISSUE THREAD

**Issue by eddardsnarf** [babagreensheep/jellyfin-tizen-docker#25]
Tried installing the tizen app throught this repo.
Everything went smoothly without any errors, until i started the app on the TV.
I'm getting 
`Unexpected Application Error!
Failed to contruct `Headers`: No matching constructor signature.
TypeError: Failed to contrcut `Headers`: No matching contructor signature.
at TypeError (native)
at ce(file:///www/node_modules@remix-run.router.010f1f9a5259d4778db5.chunk.js:2:23231)
`

Any help would be much appreciated!

**Comment by eddardsnarf** [comment]
Minor update.
I used the `latest` build from :https://github.com/jeppevinkel/jellyfin-tizen-builds
and just did the tizen install step in this docker. 
It worked like a charm.

Not sure how to fix the above error though.

**Comment by livyk** [comment]
@eddardsnarf 
It seems that the latest master of the `jellyfin-web` doesn't work (in our cases 😅). You should get the latest released tag (v10.8.13) instead by updating the `Dockerfile` a bit
```
RUN git clone https://github.com/jellyfin/jellyfin-web.git /home/jellyfin/jellyfin-web
```
with
```
RUN git clone --depth 1 --branch v10.8.13 https://github.com/jellyfin/jellyfin-web.git /home/jellyfin/jellyfin-web
```

**Comment by SergioPN** [comment]
> @eddardsnarf It seems that the latest master of the `jellyfin-web` doesn't work (in our cases 😅). You should get the latest released tag (v10.8.13) instead by updating the `Dockerfile` a bit
> 
> ```
> RUN git clone https://github.com/jellyfin/jellyfin-web.git /home/jellyfin/jellyfin-web
> ```
> 
> with
> 
> ```
> RUN git clone --depth 1 --branch v10.8.13 https://github.com/jellyfin/jellyfin-web.git /home/jellyfin/jellyfin-web
> ```

This is correct. Tried with the latest master and got the Headers error. 10.8.13 worked just fine.
